flightmonitor/static/js/common.js

```diff
@@ -16,7 +16,6 @@ var SETMODE_CONST = '<select id = "mode">' +
     '<input type="submit" value="SET MODE">' +
     '</form>';
 
-var TELEMETRY_CATEGORIES = ["GLOBAL_POSITION_INT","BATTERY_STATUS"]
```

Comment:
If the example updateTelemetryFields is no longer needed, can you remove that from the code? (lines 31-33)